backend/src/migrations/m0041_fix_token_module_reject_reasons.sql

```diff
@@ -0,0 +1,169 @@
+-- Migration to fix previously stored token module reject reasons
```

Comment:
This whole migration seems rather complex and not sure if we have all edge cases. To avoid these old to new type parsing (which is hard to review).

I wonder if we can generate much easier readable/reviewable code. My idea would be to identify all the `TokenUpdateTransactionFailed` tx and then parse the value from the below column through our cbor decoding function from the SDK and then overwrite what ever we need overwrite in the tables. Hope that would avoid to even bother what the `old/wrong values` were in the tables. 
https://github.com/Concordium/concordium-scan/blob/main/backend/src/migrations/m0001-initialize.sql#L224